### DISCUSSION TITLE
How do I build a Xyna Docker image?

### DISCUSSION THREAD

**Discussion by johannesheucher-gip** [Xyna-Factory/xyna-factory#345] [Q&A] [answered]
For my Docker environment I need Xyna Factory to run inside a Docker Container. Will Xyna Factory also be published as a Docker Image or will a Dockerfile be published via GitHub?

**Comment by LukasFey-GIP** [comment, marked as answer]
Docker images are published on dockerhub: https://hub.docker.com/u/xynafactory
You can find additional information, as well as a guide, in the wiki: https://github.com/Xyna-Factory/xyna/wiki/Easy-installation-via-Docker
If you want to create your own image and use a xyna image as a base, use https://hub.docker.com/r/xynafactory/xynabase.
